rpc/client.go

```diff
@@ -16,12 +21,19 @@ import (
 )
 
 // UploadReportOptions contains all the required options for uploading a report
-// to both Cedar and the Data Pipes service.
+// to both Cedar and SPS.
+
 type UploadReportOptions struct {
 	Report          *poplar.Report
-	ClientConn      *grpc.ClientConn
+	CedarClientConn *grpc.ClientConn
+	SPSClientConn   *http.Client
+	SPSUrl          string
 	SerializeUpload bool
 	DryRun          bool
+
+	SendToCedar  bool //Whether to send the report to Cedar.
+	SendToSPS    bool //Whether to send the report to SPS.
+	SendRatioSPS int  //What fraction of requests to send to SPS. The format will be 1/x, where x is the number set here.
```

Comment:
These are the 5 new variables that will need to be set in the codebase. We can either hardcode them and deal with them as commits when we change them or they can integrated in as actual configuration values. The latter would be preferable, but I'm unsure how quickly the hosts will pick up on the new configuration when it is updated in the database.